src/components/user/SignupContainer.js

```diff
@@ -42,15 +29,29 @@ class SignupContainer extends React.Component {
   }
 
   render() {
-    const { handleSubmit, handleSignupSubmission, pristine, submitting, renderTextField, renderCheckbox } = this.props;
+    const { handleSubmit, handleSignupSubmission, pristine, submitting, renderTextField, renderCheckbox, content } = this.props;
     const { formatMessage } = this.props.intl;
-    return (
+    const localMessages = content?.title ? {
+      title: { id: 'user.signup.title', defaultMessage: content.title },
+      intro: { id: 'user.signup.intro', defaultMessage: content.description },
+      userEmail: { id: 'email.label', defaultMessage: content.emailLabel },
+      userFullName: { id: 'user.fullName.label', defaultMessage: content.fullNameLabel },
+      userPassword: { id: 'user.password.label', defaultMessage: content.passwordLabel },
+      userConfirmPassword: { id: 'user.confirm.password.label', defaultMessage: content.confirmPasswordLabel },
+      userNotes: { id: 'user.notes.label', defaultMessage: content.notesLabel },
+      notesHint: { id: 'notes.hint.label', defaultMessage: content.notesHint },
+      userSignup: { id: 'user.signup.label', defaultMessage: content.signUpButton },
+      userConsent: { id: 'user.consent.label', defaultMessage: serializeSlateToHtml(content.consentLabel) },
+
+    } : null;
+
+    return (localMessages ? (
```

Comment:
To reduce amount of changes, why not keep the name `messages` instead of `localMessages`?

Reply:
Sure I can go that 👍